src/proto/tablet.proto

```diff
@@ -242,14 +242,13 @@ message ScanRequest {
     optional uint32 pid = 6;
     optional bool enable_remove_duplicated_record = 7 [default=false];
     optional string idx_name = 8;
-    optional string ts_name = 9;
-    optional GetType st_type = 10 [default = kSubKeyLe];
-    optional GetType et_type = 11 [default = kSubKeyGt];
-    optional bool enable_remove_ts = 12 [default = false];
-    optional uint32 atleast = 13 [default = 0];
-    repeated uint32 projection = 14;
-    repeated uint32 pid_group = 15;
-    optional bool use_attachment = 16 [default = false];
+    optional GetType st_type = 9 [default = kSubKeyLe];
```

Comment:
The unique numbered tags changed, do we consider about the backward compatibility?

Reply:
Do not consider compatibility